parastell.py

```diff
@@ -119,33 +128,33 @@ def legacy_export():
         
         # DAGMC export
         cubit.cmd(
-            f'export dagmc "dagmc.h5m" {facet_tol_str} {len_tol_str} '
+            f'export dagmc "{dir}{filename}.h5m" {facet_tol_str} {len_tol_str} '
```

Comment:
The best way to do this is use the python `Path` packages and form a fully qualified file path